spec/lib/bad_list_subscriptions_migrator_spec.rb

```diff
@@ -1,75 +1,100 @@
 RSpec.describe BadListSubscriptionsMigrator do
-  let(:prefix) { "topic" }
+  describe "#process_all_lists" do
+    let(:prefix) { "topic" }
 
-  let(:child_benefit_base_params) do
-    {
-      title: "Child Benefit",
-      url: "/topic/benefits-credits/child-benefit",
-    }
-  end
+    let(:child_benefit_base_params) do
```

Comment:
minor: it'd be nice if this didn't show up in the diff but not bothered if we want to prioritise getting this out

Reply:
If you click to the settings cog at the top (to the right of Conversations), there's a checkbox for hide whitespace!

Reply:
Yeah I used that, more thinking as a cleaning up of git blame

Reply:
Once we've run this task I'll be removing the whole file, so if it's OK with you I might save myself the rebase wrangling.